.github/workflows/ci.yml

```diff
@@ -120,7 +126,7 @@ jobs:
 
       - uses: Swatinem/rust-cache@v1
         with:
-          key: fast-MSRV
+          key: v0-fast-MSRV
```

Comment:
Would it be enough to just wait for the next time we touch Cargo.toml or Cargo.lock, which should be within the next few days?

Reply:
sure. I just want to verify this is actually working. I can revert that change